src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public class Main {
+    public static void main(String[] args) {
+        FigureSupplier figureSupplier = new FigureSupplier();
+        int arrayNumber = 7;

```

Comment:
In the main() method we need to create an array of figures (the size of array can be 3 or 6, it doesn't matter). The first half of figures in this array should be generated with random parameters.

You need create array Figure[], fill it, and after that call draw on the each array element

Reply:
Not fixed.
It should look something like this:

Reply:
```suggestion
    public static void main(String[] args) {
        int arrayNumber = 7;
        Figure[] figures = new Figure[arrayNumber];
        FigureSupplier figureSupplier = new FigureSupplier();
        for (int i = 0; i < figures.length; i++) {
            if (i < figures.length / 2) {
                firures[i] = figureSupplier.getRandomFigure();
            } else {
                figures[i] = figureSupplier.getDefaultFigure();
            }
        }
        
        for (Figure figure : figures) {
            System.out.println(figure.draw());
        }
```